src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/FilterUIHintAttribute.cs

```diff
@@ -24,7 +24,7 @@ public sealed class FilterUIHintAttribute : Attribute
         /// Gets the name of the presentation layer that supports the control type
         /// in <see cref="FilterUIHint"/>
         /// </summary>
-        public string PresentationLayer => _implementation.PresentationLayer;
+        public string? PresentationLayer => _implementation.PresentationLayer;
```

Comment:
Should second generic arg be nullable for `ControlParameters`?

Reply:
Good catch, thanks - this indeed fails compilation when nullability checks are enabled (see build hack in the original post).